test/test_codec.py

```diff
@@ -24,72 +25,72 @@ def test_bitstruct_format(self):
         s1 = Signal('signal')
         self.assertEqual(s1.bitstruct_format(), '<s0')
 
-        s2 = Signal('signal', signalSize=8)
+        s2 = Signal('signal', size=8)
         self.assertEqual(s2.bitstruct_format(), '<s8')
 
-        s3 = Signal('signal', signalSize=8, is_signed=False)
+        s3 = Signal('signal', size=8, is_signed=False)
         self.assertEqual(s3.bitstruct_format(), '<u8')
 
-        s4 = Signal('signal', signalSize=8, is_little_endian=False)
+        s4 = Signal('signal', size=8, is_little_endian=False)
         self.assertEqual(s4.bitstruct_format(), '>s8')
 
     def test_encode_signal(self):
```

Comment:
I just realized that you copied as opposed to moving the test.  Do move instead, so delete this copy I think.  Ditto for decode.  That'll make it easier to keep track of what is left in the old test world.